PopupMenu/PopupMenu.js

```diff
@@ -0,0 +1,90 @@
+import compose from 'ramda/src/compose';
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Slottable from '@enact/ui/Slottable';
+import Transition from '@enact/ui/Transition';
+
+import Skinnable from '../Skinnable';
+import Heading from '../Heading';
+import IconButton from '../IconButton';
+
+import PopupState from '../Popup/PopupState';
+
+import componentCss from './PopupMenu.module.less';
+
+const PopupMenuBase = kind({
+	name: 'PopupMenu',
+	propTypes: {
+		closeButton: PropTypes.bool,
+		css: PropTypes.object,
+		noAnimation: PropTypes.bool,
+		onCloseButtonClick: PropTypes.func,
+		onHide: PropTypes.func,
+		open: PropTypes.bool,
+		title: PropTypes.string
+	},
+
+	defaultProps: {
+		noAnimation: false,
+		closeButton: false,
+		open: false
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'popupMenu'
+	},
+
+	render: ({children, closeButton, css, noAnimation, onCloseButtonClick, onHide, open, title, ...rest}) => {
+		return (
+			<Transition
+				noAnimation={noAnimation}
+				visible={open}
+				direction="down"
+				duration="short"
+				type="fade"
+				className={css.popupTransitionContainer}
+				onHide={onHide}
+				css={css}
+			>
+				<Layout orientation="vertical" align="center center" {...rest}>
```

Comment:
Are you using `Layout` instead of `Column` for possible future horizontal support?

Reply:
That was what I was thinking, but I didn't have a practical reason to arbitrarily switch the orientation in one skin or another, so I left it hardcoded at the moment with capability for override by `{...rest}`.

Fun fact: `Layout` is one less function call than `Column`, since `Column` is just a function that sets the `orientation`, so it's a tiny bit more efficient.